README.md

```diff
@@ -41,17 +41,6 @@ In the File Sharing tab, click '+' icon and select `/files` (the directory you c
 
 ![screen shot 2017-02-09 at 12 49 54 am](https://cloud.githubusercontent.com/assets/720709/22776939/80c84282-ee66-11e6-9f6a-6d128af6ca06.png)
 
-Then, you need to comment out the first 4 lines of quickstart.sh or quickstartdev.sh
-
-These four lines needs to be commented out.
-
-```
- #docker-machine create --driver virtualbox default
```

Comment:
yeah, they need to uncomment these lines. Maybe we can change the wording for that. Saying if you are using older mac or windows for your host, you need to uncomment these lines.